package/yast2-vm.changes

```diff
@@ -1,5 +1,13 @@
 -------------------------------------------------------------------
-Fri Sep 01 19:57:03 UTC 2023 - Josef Reidinger <jreidinger@suse.com>
+Thu Apr 20 15:22:54 MDT 2023 - carnold@suse.com
+
+- The libvirtd daemon is deprecated and is going away.
+  The replacements are virtqemud for KVM/qemu and virtxend for Xen.
+  (bsc#1210572)
+- 4.6.1
+
+-------------------------------------------------------------------
+Fri Mar 03 14:44:07 UTC 2023 - Ladislav Slezák <lslezak@suse.cz>
```

Comment:
Is changing the author OK?

Reply:
yes, as lslezak did original version bump. So I have no problem with it.